loader/loader.go

```diff
@@ -528,10 +528,17 @@ func (l *Loader) Init(ctx context.Context) (err error) {
 			break
 		}
 	}
+
 	if !hasSQLMode {
-		lcfg.To.Session["sql_mode"] = l.cfg.LoaderConfig.SQLMode
+		sqlModes, err3 := utils.AdjustSQLModeCompatible(l.cfg.LoaderConfig.SQLMode)
+		if err3 != nil {
+			l.logger.Warn("cannot adjust sql_mode compatible", log.ShortError(err3))
+		}
+		lcfg.To.Session["sql_mode"] = sqlModes
```

Comment:
If `err3 != nil`, `lcfg.To.Session["sql_mode"]` will be set to an empty string "". Is that expected?
If so, pls put a comment here.

Reply:
I think it is. Added in [`c50551a`](https://github.com/pingcap/dm/pull/1869/commits/c50551a94c60658f17abd0d18a5082a01d26bd38)